src/routes/docs/tooling/command-center/keyboard-shortcuts/+page.markdoc

```diff
@@ -0,0 +1,64 @@
+
+# Shortcuts {% #shortcuts %}
+
+The Appwrite Console supports keyboard shortcuts that make it easier to navigate and perform common actions quicker. The shortcuts use the following pattern: use the first letter from the call to action followed by the resource, product, service, or page you're targeting.  For example, the shortcut keys `G` + `S` navigates to the project's Storage screen. Similarly, the shortcut `G` + `F` navigates to the project's Functions screen. However, there are some exceptions. The shortcut `G` + `E` navigates to project Settings, because `G` + `S` is in use for Storage. This pattern is used to make usage of shortcuts consistent and predictable.
+
+## Global shortcuts {% #global-shortcuts %}
+
+Developers can also utilize global shortcuts anywhere in the Appwrite Console, allowing instant access to these features from any page on the Console.
+
+|PC (Windows / Linux)|macOS|Action|
+|----|----|----|
+|`Ctrl` + `K`|`⌘` + `K`|Access Command Center|
+
+
+|Shortcut|Action|
+|----|----|
+|Set theme to light|`T` then `L`|
+|Set theme to dark|`T` then `D`|
+|Set theme to dark|`T` then `A`|
+|Open Appwrite Assistant|`A` then `I`|
+
+## Project shortcuts {% #project-shortcuts %}
+
+Within the context of a project, developers can utilize project shortcuts. These shortcuts are automatically enabled once a project is selected on the Console. These shortcuts allow quick access to the project's database, storage, messaging, auth, settings, and overview pages.
+
+|Shortcut|Action|
+|----|----|
+|Go to Auth|`G` then `A`|
+|Go to Storage|`G` then `S`|
+|Go to Databases|`G` then `D`|
+|Go to Functions|`G` then `F`|
+|Go to Projects|`G` then `P`|
+|Go to Overview|`G` then `O`|
+|Go to Messaging|`G` then `M`|
+|Go to Settings|`G` then `E`|
+
+# Primary Actions
+
+Each screen on the console has a concept of a primary action. 
+For example, in the Functions screen, the primary action would be creating a function, in the Databases screen, it would be creating a database and so on. 
+The primary actions are usually triggered using the `C` key as seen in the table below. 
+
+|Shortcut|Action|
+|----|----|
+|Create project (on projects screen)| `C` then `P`|
+|Create organization|`C` then `O`|
+|Create API key (on overview screen)|`C` then `K`|
+|Create database (on database screen)|`C` then `A`|
+|Create collection (on collection screen)|`C`|
+|Create document (on documents screen)|`C` then `D`|
+|Create attribute (on documents screen)|`C` then `T`|
+
+# Accessibility  {% #accessibility %}
+
+We are committed to making the Appwrite Console accessible to all developers by following the AA-level standards of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels). Following these standards ensures that the Console is usable by people with visual, auditory, physical, speech, cognitive, language, learning, and neurological disabilities. 
```

Comment:
Can you find a way to transition into this topic from keyboard shortcuts? As of now, this section seems a little random. Something like the second paragraph might work better.